src/fragments/lib/push-notifications/ios/getting_started/30_create_app_delegate.mdx

```diff
@@ -0,0 +1,53 @@
+## Application Delegate
+
+APNs requires an application delegate to handle push notifications functionality via delegate methods. If your app does not already have an application delegate, you will need to create one:
+
+1. Create a class that subclasses `NSObject` and implements the `UIApplicationDelegate` (iOS) or `NSApplicationDelegate` (macOS) protocol.
```

Comment:
where would I create this class?

Reply:
It can be anywhere.  It's often at the top level of the application's source code, but it really depends on how the developer chooses to organize their code.